src/frontend-samples/global-display-sample/GlobalDisplayApp.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import { BingLocationProvider, queryTerrainElevationOffset, ScreenViewport } from "@bentley/imodeljs-frontend";
+import "common/samples-common.scss";
+
+export default class GlobalDisplayApp {
+  private static _locationProvider?: BingLocationProvider;
+
+  /** Provides conversion from a place name to a location on the Earth's surface. */
+  public static get locationProvider(): BingLocationProvider {
+    return this._locationProvider || (this._locationProvider = new BingLocationProvider());
+  }
+
+  /** Given a place name - whether a specific address or a more freeform description like "New Zealand", "Ol' Faithful", etc -
+   * look up its location on the Earth and, if found, use a flyover animation to make the viewport display that location.
+   */
+  public static async travelTo(viewport: ScreenViewport, destination: string): Promise<boolean> {
+    // Obtain latitude and longitude.
+    const location = await this.locationProvider.getLocation(destination);
+    if (!location)
+      return false;
+
+    // Determine the height of the Earth's surface at this location.
+    const elevationOffset = await queryTerrainElevationOffset(viewport, location.center);
+    if (elevationOffset !== undefined)
+      location.center.height = elevationOffset;
+
+    // "Fly" to the location.
+    await viewport.animateFlyoverToGlobalLocation(location);
```

Comment:
ExtonCampus model worked, but the initial load was in the south of Atlantic

Reply:
> ExtonCampus model worked, but the initial load was in the south Atlantic

@williamkbentley the sample uses a customModelList containing only the metrostation iModel. It is not intended for use with non geolocated iModels (how did you change the iModel used?)

Reply:
I edited the SampleSpec to use Exton. I'm wondering why metrostation is not geolocated.

Reply:
Metrostation **is** geolocated. That's why you can see the map. I have not seen this error. Do you have specific repro steps?

Reply:
Rebuilt and this is no longer happening

Reply:
Spoke too soon. I can reproduce this error with several landmarks around Philadelphia:
- Independence Hall
- Lincoln Financial Field
- Drexel University

I think all other searches have been successful.

Reply:
@RBBentley

Reply:
The iModel has an ECEF, so it is geolocated. However, it does not have a GCS. GlobeAnimator discovers this only after an attempt to use the GCS fails. A subsequent attempt produces the exception.

Reply:
@pmconne Could you send me the file(s) where this can be reproduced? @williamkbentley says you have local copies.

Reply:
I have versions of the metrostation file. I repro'ed using the sample showcase's version. If @williamkbentley can provide a link to that iModel on the hub, you can download it using snapshot-creator.

Reply:
I added Mark to the Metrostatin Sample that the showcase uses.

Reply:
@pmconne Some locations in the iModel have GCS when queried; other locations in the iModel do not.  'view globe location Philadelphia' returns no GCS.  'view globe location Seattle' returns a GCS.  @RBBentley do you have any idea why this would be happening?  Could this be an issue with the geoservice?

I see this code in BackgroundMapGeometry.ts tries to determine whether to try to use GCS or not:
`
  public async cartographicToDbFromGcs(cartographic: Cartographic, result?: Point3d): Promise<Point3d> {
     ...
    return this.cartesianRange.containsPoint(db) ? this._iModel.cartographicToSpatialFromGcs(cartographic) : db;
  }
`

Reply:
@RBBentley That check for `this.cartesianRange.containsPoint(db)` in the function above is causing certain locations to go down the GCS path and others not in this iModel.  When I navigate to Philadelphia, where it gives a GCS exception, the `cartesianRange()` check is passing, making the code call `this._iModel.cartographicToSpatialFromGcs(cartographic)`.  When I navigate to Seattle, where we get no GCS exception, the `cartesianRange()` check is failing and the function just returns the `db` variable.

Reply:
The following PR has merged into master, which resolves this GCS exception problem: https://github.com/imodeljs/imodeljs/pull/970

Reply:
No longer happening with v2.14.0